torch/csrc/Module.cpp

```diff
@@ -750,6 +751,7 @@ static PyMethodDef TorchMethods[] = {
   {"_standard_gamma",  (PyCFunction)THPModule_standard_gamma,    METH_VARARGS | METH_KEYWORDS, NULL},
   {"_dirichlet_grad", (PyCFunction)THPModule_dirichlet_grad,    METH_VARARGS | METH_KEYWORDS, NULL},
   {"bernoulli",       (PyCFunction)THPModule_bernoulli,         METH_VARARGS | METH_KEYWORDS, NULL},
+  {"poisson",         (PyCFunction)THPModule_poisson,           METH_VARARGS | METH_KEYWORDS, NULL},
```

Comment:
Let's avoid adding more of the sampling methods to the global namespace. `torch.distributions` is the way to expose those samplers